src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new Random().nextInt(100), new ColorSupplier().getRandomColor());
```

Comment:
The same as previous comment.
Do we really need to create a new instance of ColorSupplier and Random classes every time when this method is called?
It is better to create a new fields as class property.